cartographer/sensor/data.h

```diff
@@ -41,6 +41,10 @@ struct Data {
     PointCloud ranges;
   };
 
+  struct FixedFramePose {
```

Comment:
Because both of the fixed_frame_pose and odometry_pose are Rigid3d, the Data constructor will have the same parameter. We can discuss it if there's other solution.